### PR TITLE
Fixes #278: Check if files_path is set before override.

### LINKED ISSUE #278: Unable to override files_path setting
**What happens**

When using the `I attach the file "<file>" to "<field>"` step it attempts to upload a file from the lighting profile directory regardless of what is configured in the project's behat.yml.

**behat.yml**

```
default:
  suites:
    default:
      paths:
        - %paths.base%/features
      contexts:
        - Drupal\FeatureContext
        - Drupal\DrupalExtension\Context\DrupalContext
        - Drupal\DrupalExtension\Context\MinkContext
        - Drupal\DrupalExtension\Context\MessageContext
        - Drupal\DrupalExtension\Context\DrushContext
  extensions:
    Behat\MinkExtension:
      default_session: goutte
      javascript_session: selenium2
      goutte: ~
      files_path: "%paths.base%/media"
```

**What I expect to happen**

I expect the configured path to be used whenever we use the attach file step.

**Cause**

`lightning.behat.inc:setFilePath()` is forcing a Lightning-relative path and by the time we hit the MinkContext implementation of uploading a file we've lost the project setting.

### PR DESCRIPTION
**Pull request by swichers**
Checks if there is already a files_path set and use that instead.

### CODE REVIEW

**Comment by balsama** [comment]
This looks good to me. Thanks. I would merge now, but I know @phenaproxima is working in the space right now. Added him as a reviewer. @phenaproxima feel free to merge.

File: tests/features/bootstrap/lightning.behat.inc [hunk 1]
```diff
@@ -401,7 +401,12 @@ class LightningSubContext extends DrupalSubContextBase {
    * @BeforeScenario
    */
   public function setFilePath() {
-    $this->minkContext->setMinkParameter('files_path', __DIR__ . '/../../files');
+
+    $existing = $this->minkContext->getMinkParameter('files_path');
+    if (empty($existing)) {
+
```
Comment (phenaproxima) [inline tests/features/bootstrap/lightning.behat.inc]: Nitpick, but I can't help myself -- can the extra line after empty($existing) be removed? Once that's done, this is good to merge.

**Review by phenaproxima — CHANGES_REQUESTED** [review]

**Comment by swichers** [comment]
@phenaproxima Done

**Review by phenaproxima — APPROVED** [review]
Nice. Thanks for putting up with me!

**Pull request merged**